packages/paste-website/src/pages/layout/media-object/index.mdx

```diff
@@ -0,0 +1,287 @@
+      edges {
+        node {
+          name
+          description
+          status
+          version
+        }
+      }
+    }
+    mdx(fields: {slug: {eq: "/media-object/"}}) {
+      fileAbsolutePath
+      frontmatter {
+        slug
+        title
+      }
+      headings {
+        depth
+        value
+      }
+    }
+  }
+`;
+
+<ComponentHeader
+  name="Media Object"
+  categoryRoute={SidebarCategoryRoutes.LAYOUT}
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/layout/media-object"
+  storybookUrl="https://twilio-labs.github.io/paste/?path=/story/layout-stack--stack"
+  data={props.data.allPasteLayout.edges}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={props.data.mdx} />
+
+<content>
+
+## Guidelines
+
+### About Media Object
+
+First created by [Nicole Sullivan](http://www.stubbornella.org/content/2010/06/25/the-media-object-saves-hundreds-of-lines-of-code/) back in 2010, the Media Object in Paste aims to serve many of the same purposes as the original concept. It's a layout pattern that you will see all across the web, on almost all the websites you will come across. We're pretty sure you'll never unsee this going forward.
+
+The concept is this; there is an image or figure positioned horizontally next to some form of content. The figure can sit on either or both sides of the content. That's it. It sounds dumb, but it's literally everywhere. Even in the top of this website with the Paste logo.
+
+If you were to draw the concept, it might look something like this:
+
+<Box as="div" backgroundColor="colorBackgroundDark" padding="space40" marginBottom="space70">
+  <MediaObject as="div">
+    <MediaFigure as="div" spacing="space40">
+      <Box backgroundColor="colorBackgroundPrimary" height="sizeIcon110" width="sizeIcon110" />
+    </MediaFigure>
+    <MediaBody as="div">
+      <Box backgroundColor="colorBackgroundPrimary" height="sizeIcon110" width="100%" />
+    </MediaBody>
+  </MediaObject>
+</Box>
+
+The aim of the Media Object to make this extremely common layout pattern trivial to achieve time and time again.
+
+#### Accessibility
+
+By default, Media Object has no accessibility concerns. While using Media Object, it is up to you to manage the resulting
+accessibility implications. You may want to consider how the Media Object affects readability due to ordering, and
+the visual or content hierarchy of your page.
+
+## Examples
+
+### Basic Media Object
+
+The Media Object is a composition of three things;
+
+- The `MediaObject`
+- A `MediaFigure`
+- A `MediaBody`
+
+The `MediaObject` acts as the outer wrapper of the pattern. The `MediaFigure` is the container for the content that is often graphical in nature and fixed in size. The `MediaBody` is a container typically for accompanying text content and fills the remaining space.
+
+The content of a `MediaFigure` and `MediaBody` is entirely up to you.
+
+<LivePreview scope={{Box, MediaObject, MediaFigure, MediaBody}} language="jsx">
```

Comment:
Add a heading in this example, it feels incomplete.

Also add some references to Black authors, or activists, or messaging